sound/soc/sof/topology.c

```diff
@@ -1738,63 +1738,18 @@ static int sof_process_load(struct snd_soc_component *scomp, int index,
 	struct sof_abi_hdr *pdata = NULL;
 	struct sof_ipc_comp_process *process;
 	size_t ipc_size, ipc_data_size = 0;
-	int ret, i, offset = 0;
+	int ret, i;
+	int ipc_cmd, ctrl_type;
```

Comment:
reverse xmas tree and one declaration per line if possible.

Reply:
ok